br/pkg/stream/rewrite_meta_rawkv.go

```diff
@@ -432,9 +432,21 @@ func (sr *SchemasReplace) rewriteValue(
 }
 
 // RewriteKvEntry uses to rewrite tableID/dbID in entry.key and entry.value
-func (sr *SchemasReplace) RewriteKvEntry(e *kv.Entry, cf string) (*kv.Entry, error) {
+func (sr *SchemasReplace) RewriteKvEntry(e *kv.Entry, cf string, insertDeleteRangeForTable InsertDeleteRangeForTable, insertDeleteRangeForIndex InsertDeleteRangeForIndex) (*kv.Entry, error) {
 	// skip mDDLJob
+
```

Comment:
whether it has some different value between default-cf and write-cf ?

Reply:
It seems that the key with prefix `mddlJobHistory` whose value can be decoded into `model.Job` uses default-cf. Maybe we can directly try to decode the value ignoring its cf.